### PR TITLE
feat: unpack query util function

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | Originated from https://github.com/ripe-tech/ripe-id-mobile/pull/22 |
| Dependencies | |
| Decisions | Parse query function (similar to RIPE SDK) |
| Animated GIF | |

### CODE REVIEW

File: js/util.js [hunk 1]
```diff
@@ -51,3 +51,37 @@ export const buildSlug = value => {
...
+
+/**
+ * Parses a query string and builds an object with each
+ * query parameter and its respective value.
+ *
+ * @param {String} query The query string to be parsed.
+ * @returns An object containing the query parameters
+ * keys and its respective values
+ */
+export const unpackQuery = query => {
```
Comment (gcandal) [inline js/util.js:63]: https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams ?

Comment (BeeMargarida) [reply js/util.js:63]: Does not work, related to this: https://github.com/facebook/react-native/issues/23922
TLDR: necessary to add a polyfill

**Review by gcandal — APPROVED** [review]

**Comment by gcandal** [comment]
@BeeMargarida the changelog for this is not as relevant because it's an internal lib, but have in mind that stuff that has other stakeholders (e.g. ripe-core, hermes-proxy, etc) needs to have a more clear description so that everyone can understand what happened.

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Comment by joamag** [comment]
@BeeMargarida check the add unit tests
